docs: Change default install to folder

cjfullerton opened the pull request:
Change defult install script commands to install the "one folder" version. Update the "What does this script do?" section to reflect changes. Closes #429.

cjfullerton committed:
docs: Change default install to folder

cjfullerton committed:
docs: Tweak wording.

aplowman reviewed (changes requested):
Looks good, but could you update the admonition section so it has the dropdown class and fix the formatting issue, so it is formatted more like in [matflow here](https://github.com/hpcflow/matflow-new/blob/65fbbc134af442393df424f7e4121ca053cc7551/docs/source/installation.rst?plain=1#L50)? ([Currently](https://hpcflow.github.io/docs/stable/installation.html), random bits of it appear in bold) Thanks!

cjfullerton committed:
docs: Make script explanation dropdown

cjfullerton committed:
docs: tidy up installation.rst

cjfullerton commented:
> Looks good, but could you update the admonition section so it has the dropdown class and fix the formatting issue, so it is formatted more like in [matflow here](https://github.com/hpcflow/matflow-new/blob/65fbbc134af442393df424f7e4121ca053cc7551/docs/source/installation.rst?plain=1#L50)? ([Currently](https://hpcflow.github.io/docs/stable/installation.html), random bits of it appear in bold) Thanks!

Added dropdown option to the installation page.

It's not clear why segments appear in bold. I have done a diff with the matflow ``installation.rst`` and adjusted formatting until only differences are the folder names. Hopefully that will sort it.
